packages/legend-application-studio/src/components/editor/editor-group/service-editor/__tests__/ServiceEditorTest.test.tsx

```diff
@@ -184,6 +183,75 @@ test(
   },
 );
 
+test(
+  integrationTest(
+    'Test Enternal Format Service Test Parameter Setup for Binding with Byte',
+  ),
+  async () => {
+    const MOCK__editorStore = TEST__provideMockedEditorStore({ pluginManager });
+    const renderResult = await TEST__setUpEditorWithDefaultSDLCData(
+      MOCK__editorStore,
+      {
+        entities: TEST_DATA__ExternalFormatServiceEntities,
+      },
+    );
+    MockedMonacoEditorInstance.getValue.mockReturnValue('');
+    await TEST__openElementFromExplorerTree(
+      'demo::externalFormat::flatdata::simple::service::FlatdataInternalizeByte',
+      renderResult,
+    );
+    const editorGroup = await waitFor(() =>
+      renderResult.getByTestId(LEGEND_STUDIO_TEST_ID.EDITOR_GROUP),
+    );
+    await waitFor(() => getByText(editorGroup, 'Test'));
+    fireEvent.click(getByText(editorGroup, 'Test'));
+    const serviceTestEditor = await waitFor(() =>
+      renderResult.getByTestId(LEGEND_STUDIO_TEST_ID.SERVICE_TEST_EDITOR),
+    );
+    await waitFor(() => getByText(serviceTestEditor, 'Setup'));
+    fireEvent.click(getByText(serviceTestEditor, 'Setup'));
+    const serviceTestSetupEditor = await waitFor(() =>
+      renderResult.getByTestId(
+        LEGEND_STUDIO_TEST_ID.SERVICE_TEST_EDITOR__SETUP__PARAMETERS,
+      ),
+    );
+    fireEvent.click(getByText(serviceTestEditor, 'Setup'));
+    const bindingParmPairsForByte = MOCK__editorStore.tabManagerState
+      .getCurrentEditorState(ServiceEditorState)
+      .testableState.selectedSuiteState?.testStates[0]?.setupState.getBindingWithParamFromQuery();
+    expect(bindingParmPairsForByte).toHaveLength(1);
+    expect(
+      guaranteeNonNullable(guaranteeNonNullable(bindingParmPairsForByte)[0])
+        .binding.name,
+    ).toBe('PersonBinding');
+    //Test showing actual string as value for byte type
+    await waitFor(() =>
+      getByText(
+        serviceTestSetupEditor,
+        'First Name,Last NameJohn,DoeOlive,Yew',
+      ),
+    );
+    const parameterValueStates =
+      MOCK__editorStore.tabManagerState.getCurrentEditorState(
+        ServiceEditorState,
+      ).testableState.selectedSuiteState?.testStates[0]?.setupState
+        .parameterValueStates;
+    expect(parameterValueStates).toHaveLength(2);
+    expect(
+      JSON.stringify(
+        parameterValueStates?.find((p) => p.parameterValue.name === 'data')
+          ?.parameterValue.value,
+      ),
+    ).toBe(
+      '{"_type":"byteArray","sourceInformation":{"sourceId":"","startLine":39,"startColumn":20,"endLine":39,"endColumn":71},"value":"Rmlyc3QgTmFtZSxMYXN0IE5hbWVKb2huLERvZU9saXZlLFlldw=="}',
```

Comment:
The test tests from two aspects

1. UI displays the actual string instead of base64string  

did it by 
`    await waitFor(() =>
      getByText(
        serviceTestSetupEditor,
        'First Name,Last NameJohn,DoeOlive,Yew',
      ),
    );`

2.  test that we stores base64String in the state 

Since we don't want to assert on state, we could just delete no.2